kamon-core/src/main/scala/kamon/metric/Gauge.scala

```diff
@@ -42,6 +62,20 @@ object Gauge {
 
     @volatile private var _currentValue = 0D
 
+    override def increment(): Gauge = increment(1)
+
+    override def increment(times: Double): Gauge = {
+      _currentValue += times
+      this
+    }
+
+    override def decrement(): Gauge = decrement(1)
+
+    override def decrement(times: Double): Gauge = {
+      _currentValue -= times
```

Comment:
This should guard against negative values, same as `update` does.

Reply:
@ivantopo should I change `_currentValue` to an `AtomicDouble`?

Reply:
Is there an AtomicDouble in the JDK?

Reply:
More or less: https://stackoverflow.com/questions/5505460/java-is-there-no-atomicfloat-or-atomicdouble